pyresample/geometry.py

```diff
@@ -296,11 +296,12 @@ def get_boundary_lonlats(self):
         return (SimpleBoundary(s1_lon.squeeze(), s2_lon.squeeze(), s3_lon.squeeze(), s4_lon.squeeze()),
                 SimpleBoundary(s1_lat.squeeze(), s2_lat.squeeze(), s3_lat.squeeze(), s4_lat.squeeze()))
 
-    def get_bbox_lonlats(self, frequency: Optional[int] = None, force_clockwise: bool = True) -> tuple:
+    def get_bbox_lonlats(self, bbox_shape: Optional[int] = None, force_clockwise: bool = True,
```

Comment:
Could we use `n_vertices`, 'n_vertices_per_side' or something like that? Using "shape" makes me think to an array shape ... or the output shape...

In this case the length of lons/lats is `4*bbox_shape`